protobuf-solidity/src/protoc/plugin/gen_decoder.py

```diff
@@ -126,7 +126,7 @@ def gen_struct_decoder(f, msg, parent_struct_name):
   )
 
 def gen_struct_decoders(msg, parent_struct_name):
-  return ''.join(list(map(
+  return ''.join(set(map(
```

Comment:
I realized that `set` provides unstable sorting, so unnecessary changes occur in the generated code.